### PR TITLE
Add platform info utilities

### PR DESCRIPTION
**Pull request by nikolai-laevskii**
Adds shorthands for getting information about platform action is running on:

```js
import { platform } from '@actions/core'

platform.platform // 'win32'
platform.arch // 'x64'
platform.isWindows // true
platform.isMacOS // false
platform.isLinux // false

const {
  name, // Microsoft Windows 11 Enterprise
  version, // 10.0.22621
} = await platform.getInfo()
```

Some checks are failing but not due changes. Tests and documentation are updated.

### CODE REVIEW

**Review by dsame — APPROVED** [review]

File: packages/helpers/README.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+
+#### Platform helper
+
+Provides shorthands for getting information about platform action is running on.
+
+```js
+import { platform } from '@actions/core'
```
Comment (MaksimZhukov) [inline packages/helpers/README.md]: this is a typo I guess
```suggestion
import { platform } from '@actions/helpers'
```

File: packages/helpers/README.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+platform.platform // 'win32'
```
Comment (MaksimZhukov) [inline packages/helpers/README.md]: nitpick: I recommend listing all values in the comment

**Review by MaksimZhukov — APPROVED** [review]

**Review by IvanZosimov — APPROVED** [review]

File: packages/helpers/package.json [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+    "github",
+    "actions",
+    "helpers"
+  ],
+  "author": "",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/actions/toolkit/issues"
+  },
+  "homepage": "https://github.com/actions/toolkit#readme",
```
Comment (IvanZosimov) [inline packages/helpers/package.json]:
```suggestion
  "homepage": "https://github.com/actions/toolkit/tree/main/packages/helpers",
```

**Comment by nikolai-laevskii** [comment]
https://github.com/actions/toolkit/pull/1551

**Pull request closed without merging**